buildSrc/src/main/kotlin/profiler.publication.gradle.kts

```diff
@@ -37,3 +38,18 @@ fun Project.gradleInternalRepositoryUrl(): URI {
     val repositoryQualifier = if (isSnapshot) "snapshots" else "releases"
     return uri("https://repo.gradle.org/gradle/ext-$repositoryQualifier-local")
 }
+
+val signArtifacts: Boolean = !System.getenv("PGP_SIGNING_KEY").isNullOrEmpty()
+tasks.withType<Sign>().configureEach { isEnabled = signArtifacts }
```

Comment:
Nit: can we add some extra lines between these code?

Reply:
Done.